flink-table/flink-sql-gateway/src/test/java/org/apache/flink/table/gateway/rest/SessionRelatedITCase.java

```diff
@@ -85,11 +115,11 @@ void testCreateAndCloseSessions() throws Exception {
             CompletableFuture<OpenSessionResponseBody> response =
                     sendRequest(openSessionHeaders, emptyParameters, openSessionRequestBody);
             String sessionHandleId = response.get().getSessionHandle();
-            assertThat(sessionHandleId).isNotNull();
+            assertNotNull(sessionHandleId);
```

Comment:
Why modify this? I think `org.assertj.core.api.Assertions.assertThat` is suggested test API.